config.json.template

```diff
@@ -11,7 +10,8 @@
   },
   "pywren": {
     "storage_bucket": "****",
-    "runtime": "ibmfunctions/pywren-metabolomics:3.6"
+    "runtime": "ibmfunctions/pywren-metabolomics:3.6",
+    "data_cleaner": true
```

Comment:
Does this mean we no longer need to put `pw.clean()` everywhere?

Reply:
yes, exactly